src/pages/home/ReportScreen.tsx

```diff
@@ -462,6 +462,7 @@ function ReportScreen({
         });
         return () => {
             interactionTask.cancel();
+            Report.unsubscribeFromReportChannel(report.reportID);
```

Comment:
Any reason we're not checking for `didSubscribeToReportTypingEvents.current` here (as you originally [proposed](https://github.com/Expensify/App/issues/43089#issuecomment-2149100121))?